src/Arriba/Arriba/Caching/IObjectCache.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Arriba.Caching
+{
+    public interface IObjectCache : IDisposable
+    {
+        object Get(string key);
```

Comment:
Isn't it better to use an interface for the key other than use string?

Reply:
Yes. I don't personally like this caching model, but did not want to make changes to it as part of the work. The goal of this change was only to be able to move the dependency into the adapter and keeping the existing behavior intact.